packages/terra-toggle-section-header/docs/README.md

```diff
@@ -0,0 +1,34 @@
+# Terra Toggle Section Header
+
+Toggle section header component that transitions content in and out with a click.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-toggle-section-header`
+
+## Usage
+
+```jsx
+import React from 'react';
+import ToggleSectionHeader from 'terra-toggle-section-header';
+
+<ToggleSectionHeader
+  title="Section Header Title"
+  level={3}
+  isInitiallyOpen
+>
+  <div>Hello, World!</div>
+</ToggleSectionHeader>
+```
+
+## Component Features
+
+ * [Cross-Browser Support](https://github.com/cerner/terra-core/wiki/Component-Features#cross-browser-support)
+ * [LTR/RTL Support](https://github.com/cerner/terra-core/wiki/Component-Features#ltr--rtl-support)
+<!-- Uncomment supported features.
```

Comment:
Is this another case where we're leaving potential future features commented out?

Reply:
I'm just going with the same convention that I have of this point of leaving it in, as the comment says to `uncomment`. That way, if things are supported in the future, they can just be moved outside the comment block

Reply:
Looks like this is having some issues rendering

Reply:
Ahh! I should have caught this, thanks @ChaseSinclair b2d53e2d